### DISCUSSION TITLE
How to assign an issue that is closed to current sprint?

### DISCUSSION THREAD

**Discussion by RonenYAtZenity** [leonsteinhaeuser/project-beta-automations#42] [Q&A] [answered]
Hi,

I am using this action for long time to assign every issue that is being opened to our Project. Thank you for it.

We are working with Iterations to allocate our 2 week sprints. I understand that I can use this action to set an iteration (our sprint). 
What I would like to do is to set on every issue that is being closed the project sprint (iteration) field to current sprint. Is it possible?

Thanks,
Ronen

**Comment by leonsteinhaeuser** [comment, marked as answer]
Hi Ronen,

unfortunately the GitHub API does not support something like `@current`. I already tested a few things and checked the API, but couldn't find a solution for this case. The only thing we could add is a custom implementation for `@current` and do that part ourselves.

Greets,
Leon

**Comment by leonsteinhaeuser** [comment]
@RonenYAtZenity I have some good news. Today I had some time to implement this feature and in my tests it worked quite well. I would like you to test the functionality on your end and give me feedback. Since the function has not yet been released, you can find all information about this function in PR: #45
